src/services/OrderService.tsx

```diff
@@ -0,0 +1,45 @@
+import { OrderDescription } from "@adapters/OrderDescription";
+import { useContext, useEffect, useState, createContext, useRef } from "react";
+import { Order } from "@models/Order";
+import { writeSync } from "fs";
+
+export interface IOrderService {
+  getOrderDescriptions(): Promise<OrderDescription[]>;
+  sendOrder(order: Order): void;
+}
+
+export const OrderContext = createContext<IOrderService>({} as IOrderService);
+
+export const OrderService = ({ children }: any) => {
+  const orderSocket = useRef(
+    new WebSocket(
+      `ws://${process.env.SERVER_IP}:${process.env.SERVER_PORT}/wsOrder`
```

Comment:
all the request to the backend should be prefixed with "backend" (e.g. "ws://localhost:1234/backend/wsOrder")